schemas/logs/apache/access_combined.yml

```diff
@@ -0,0 +1,41 @@
+schema: Apache.AccessCombined
+parser:
+  native:
+    name: Apache.AccessCombined
+description: Apache HTTP server access logs using the 'combined' format
+referenceURL: https://httpd.apache.org/docs/current/logs.html#combined
+version: 0
+fields:
```

Comment:
I assume these are all auto-generated, is it possible to capitalize the first letter of each field? e.g.
```
Schema: Apache.AccessCombined
Parser:
  Native:
    Name: Apache.AccessCombined
Description: Apache HTTP server access logs using the 'combined' format
ReferenceURL: https://httpd.apache.org/docs/current/logs.html#combined
Version: 0
Fields:
```

This would be aligned with how we format every other CLI configuration file for Panther.

Reply:
Not really possible now unless we delay this by a lot. We would need to have case-insensitive parsing for the schemas and change the JSONSchema rules. I think this will be left like this until we go to a `version: 1`.

Reply:
@nhakmiller note that this is the current logschema format used in custom logs: https://docs.runpanther.io/data-onboarding/custom-log-types#writing-a-log-schema-for-json-logs

Reply:
Can I open an issue in Panther to change that, or are you committed to the way it is now? Important to remember that unlike the panther repo this is a heavily end user facing repo, where we can provide a better user experience we should strive to do so.

Reply:
> Can I open an issue in Panther to change that, or are you committed to the way it is now? Important to remember that unlike the panther repo this is a heavily end user facing repo, where we can provide a better user experience we should strive to do so.

It is quite more complex to achieve this because of the YAML specs already stored in DDB. We would also have to update the whole JSONSchema that validates specs and change and re-test much of the customlogs-related pipeline. I would prefer to defer this for when we have time to do all that.